### DISCUSSION TITLE
deepin-editor Text wrapping doesn't work correctly

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#3383] [Features Request & Ideas | 特性请求 & 头脑风暴]
As you can see on the video a `hello` word shouldn't be cut in half, but jump entirely to the bottom line.

https://user-images.githubusercontent.com/39202451/184941950-f2481936-8b62-47f2-8370-654de76d50e7.mp4

**Comment by BLumia** [comment]
@meiyixiang 需求：用户希望编辑器可以按词折行而不是按字符。
